Entities/Structures/Trampoline/TrampolineLogic.as

```diff
@@ -138,6 +139,10 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid, Vec2f normal, Vec2f point
 			velocity.RotateBy(angle);
 
 			blob.setVelocity(velocity);
+			
+			blob.server_Hit(blob, blob.getPosition(), blob.getVelocity(), 0.0f, Hitters::fall_trampoline, true);
```

Comment:
Not 100% sure about this, some onHit hooks may react differently based on parameters. It would need checking for edge cases.

Reply:
I will check onHit() on every blob. If there is anything awry, we can still resort to using the PR without the Trampoline.

Reply:
The only thing I found is Bison will run `MadAt()` when touching your trampoline. Should be easy to fix.

Reply:
I found that watersplash will also cause Bison to become mad.
Should I add a check if damage is above 0? Or should I check for `Hitters::trampoline` specifically?

Reply:
I would check if damage is above 0